platform/javascript/js/libs/library_godot_audio.js

```diff
@@ -250,7 +250,7 @@ const GodotAudioWorklet = {
 					GodotAudioWorklet.worklet = null;
 					GodotAudioWorklet.promise = null;
 					resolve();
-				}).catch(function(err) { /* aborted? */});
+				}).catch(function (err) { /* aborted? */ });
```

Comment:
This should be amended in the first commit if we want to be thorough ;)

Reply:
righty, hasty me, thanks for cheking